sklearn/model_selection/_search_successive_halving.py

```diff
@@ -992,6 +1006,8 @@ class HalvingRandomSearchCV(BaseSuccessiveHalving):
     The parameters selected are those that maximize the score of the held-out
     data, according to the scoring parameter.
 
+    All parameter combinations scored with a NaN will share the lowest rank.
```

Comment:
Is this what you had in mind in terms of doc string update? I wasn't quite sure where to add this :-/ (same for the other `Halving*CV`)

Reply:
Adding to the notes works for me.